### PR TITLE
46+47+48

### LINKED ISSUE #47: Improve templates for deployment scripts
It would be nice to have deploy/configuration scripts with some template prefixes like `application.conf.example` and add actual scripts to `.gitignore` so one does not need to revert changes of scripts before commit/push etc and does not accidentally deploy app with wrong parameters.

Another way to do this – parametrize deployment scripts. But I think the first way is better.

### PR DESCRIPTION
**Pull request by rozhok**
WIP for #46 
Fixes #47 and #48

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -13,7 +13,7 @@ db.default.password=password
 notifications.from="Sundial <sundial@yourdomain.com>"
 notifications.mode=email
 
-companion.tag=
+companion.tag="069087730660.dkr.ecr.eu-west-1.amazonaws.com/sundial-companion:1471207530"
```
Comment (nemo83) [inline conf/application.conf]: this looks like an account specific ecr setting. I would recommend deleting it.

File: deploy.sh [hunk 2]
```diff
@@ -1,24 +1,27 @@
 #!/bin/sh
 
 #Change these to match your environment
-REGION=us-east-1 # do not change this since ECR is available only in us-east-1
-DOCKER_REGISTRY=
-DOCKER_REPO=svc-sundial
+REGION=eu-west-1
+DOCKER_REGISTRY=069087730660.dkr.ecr.eu-west-1.amazonaws.com
```
Comment (nemo83) [inline deploy.sh]: ditto

File: deploy.sh [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
+DOCKER_REPO=sundial
 # An s3 bucket for deployment assets
-CODEDEPLOY_S3_BUCKET=
+CODEDEPLOY_S3_BUCKET=movedin-dev-codedeploy
```
Comment (nemo83) [inline deploy.sh]: same here

Comment (rozhok) [inline deploy.sh]: Yep, I've accidently commited actual scripts. Removed now.

**Comment by kevinoriordan** [comment]
Thank you for the PR. Good to see you're still using it.

**Comment by kevinoriordan** [comment]
Let me look over the cfn-eco.json before merging. There's a lot of duplication between that and the main template and we may be able to get around that by using nested parameterized templates.

**Comment by rozhok** [comment]
@kevinoriordan 
>There's a lot of duplication between
Yeah, it's WIP, I tried to put eco-mode into main template but it's a bit tricky so I leave it as reduced copy at the moment, also there should be added scripts in launch configuration to create record set for sundial service instance id which I not added yet.

**Pull request closed without merging**